app/src/app.config.json.tpl

```diff
@@ -138,6 +138,811 @@
+      "label": "OpenOffice 1.0/StarOffice6.0 Calc 6.0 Template"
+    },
+    {
+      "value": "application/vnd.sun.xml.draw",
+      "label": "OpenOffice 1.0/StarOffice6.0 Draw 6.0"
+    },
+    {
+      "value": "application/vnd.sun.xml.impress",
+      "label": "OpenOffice 1.0/StarOffice6.0 Impress 6.0"
+    },
+    {
+      "value": "application/vnd.sun.xml.impress.template",
+      "label": "OpenOffice 1.0/StarOffice6.0 Impress 6.0 Template"
+    },
+    {
+      "value": "application/vnd.sun.xml.writer",
+      "label": "OpenOffice 1.0/StarOffice6.0 Writer 6.0"
+    },
+    {
+      "value": "application/vnd.sun.xml.writer.template",
+      "label": "OpenOffice 1.0/StarOffice6.0 Writer 6.0 Template"
+    },
+    {
+      "value": "image/png",
+      "label": "PNG Image"
+    },
+    {
+      "value": "image/x-raw-panasonic",
+      "label": "Panasonic RAW Image"
+    },
+    {
+      "value": "image/x-raw-pentax",
+      "label": "Pentax RAW Image"
+    },
+    {
+      "value": "image/x-portable-pixmap",
+      "label": "Pixmap Image"
+    },
+    {
+      "value": "text/plain",
+      "label": "Plain Text"
+    },
+    {
+      "value": "image/x-portable-bitmap",
+      "label": "Portable Bitmap"
+    },
+    {
+      "value": "application/postscript",
+      "label": "PostScript"
+    },
+    {
+      "value": "application/remote-printing",
+      "label": "Printer Text File"
+    },
+    {
+      "value": "video/quicktime",
+      "label": "Quicktime Video"
+    },
+    {
+      "value": "video/x-rad-screenplay",
+      "label": "RAD Screen Display"
+    },
+    {
+      "value": "application/x-rar-compressed",
+      "label": "RAR Archive"
+    },
+    {
+      "value": "image/x-raw-red",
+      "label": "RED RAW Image"
+    },
+    {
+      "value": "image/x-rgb",
+      "label": "RGB Image"
+    },
+    {
+      "value": "application/rss+xml",
+      "label": "RSS"
+    },
+    {
+      "value": "image/x-cmu-raster",
+      "label": "Raster Image"
+    },
+    {
+      "value": "text/richtext",
+      "label": "Rich Text"
+    },
+    {
+      "value": "application/rtf",
+      "label": "Rich Text Format"
+    },
+    {
+      "value": "video/x-sgi-movie",
+      "label": "SGI Video"
+    },
+    {
+      "value": "text/sgml",
+      "label": "SGML (Human Readable)"
+    },
+    {
+      "value": "application/sgml",
+      "label": "SGML (Machine Readable)"
+    },
+    {
+      "value": "image/svg+xml",
+      "label": "Scalable Vector Graphics Image"
+    },
+    {
+      "value": "application/x-sh",
+      "label": "Shell Script"
+    },
+    {
+      "value": "application/x-shockwave-flash",
+      "label": "Shockwave Flash"
+    },
+    {
+      "value": "image/x-raw-sigma",
+      "label": "Sigma RAW Image"
+    },
+    {
+      "value": "image/x-raw-sony",
+      "label": "Sony RAW Image"
+    },
+    {
+      "value": "application/vnd.stardivision.chart",
+      "label": "StaChart 5.x"
```

Comment:
I guess typo here? Should be "Star" not "Sta"

Reply:
could have been Star, but I took reference from share, and its 'Sta' there

Reply:
hmm understood. I just thought about "Star" because I can see that other values have "Star" not "Sta" so it will be more consistent. @arohilaGL @MichalKinas what do you think?

Reply:
changed to Star